### PR TITLE
Enhanced gates

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
Enhanced durability research for gates against multiple raiders
Add debug for visitors getting to zero pos

Review please

### CODE REVIEW

**Comment by ravenbuilder934** [comment]
Add debug for visitors getting to zero pos? Does this close #5811?

**Comment by someaddons** [comment]
no that should just help finding the cause

**Comment by ravenbuilder934** [comment]
Oh, okay

File: src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java [hunk 1]
```diff
@@ -688,4 +689,15 @@ protected void dropInventory()
             }
         }
     }
+
+    // TODO:REMOVE DEBUG
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java:693]: todo, remove debug?

Comment (someaddons) [reply src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java:693]: ye its debug output to find the issue causing visitors dying at zero positions and is to be removed in the future xD

**Review by Mekle001 — APPROVED** [review]

**Pull request merged**